### ISSUE TITLE
Option block delete files NAS serveur

### ISSUE THREAD

**Issue by JeedHome44** [nova-video-player/aos-AVP#421]
Can you add an option to avoid deleting files in a NAS serveur?

If I connect Nova with a profil with only Play right, I can't have picture in movie.

I want to have pictures in movies but block delete option.

Thank you

**Comment by lars18th** [comment]
Hi,

I feel you're requesting for three different functionalities:
1. When connecting with a READ-ONLY network share, enable full support. At time, some glitches appear when the filesystem is readonly.
2. When connecting with a READ-WRITE network share, add a configuration value (for the share, at folder level) to disable the delete functionality in the UI (the button may be grayed out). Then the files and metadata can be updated (as the filesystem is writable), but the user can't delete content.
3. When connecting with a READ-WRITE network share, add support for a non-deletable content. This can be supported with a new entry inside the ".FILENAME.archos.resume.xml" metadata file. This flag can provide a method to block the deletion of the file called "FILENAME".

I hope the developers will want to support this three enhancements.
Regards.

**Comment by JeedHome44** [comment]
yes, it would save my family from being able to delete movies by mistake
